client-app/src/examples/news/NewsPanelModel.ts

```diff
@@ -9,8 +9,12 @@ import {newsPanelItem} from './NewsPanelItem';
 export class NewsPanelModel extends HoistModel {
     SEARCH_FIELDS = ['title', 'text'];
 
+    @bindable.ref
+    newsErrorThrown: object;
```

Comment:
- Don't think `object` is the right type.  Also -- a nit -- `error` is probably sufficient for a name here.  `news` and `thrown` implied.